docs/devel/running-locally.md

```diff
@@ -68,6 +69,12 @@ You need an [etcd](https://github.com/coreos/etcd/releases) in your path, please
 
 You need [go](https://golang.org/doc/install) in your path (see [here](development.md#go-versions) for supported versions), please make sure it is installed and in your ``$PATH``.
 
+#### OpenSSL
+
+You need [OpenSSL](https://www.openssl.org/) installed.  If you do not have the `openssl` command available, you may see the following error in `/tmp/kube-apiserver.log`:
+
+```server.go:333] Invalid Authentication Config: open /tmp/kube-serviceaccount.key: no such file or directory```
```

Comment:
This doesn't look right when I view the rendered markdown. I think you want to put the three backticks on their own line.